[MTM-50161] Review docs for 10.16 (PlaMa UI; part of MTM-47816) (#728)
SoftwareAG/c8y-docs: content/users-guide/device-management-bundle/viewing-devices.md

1. Hover over the row of the device you want to delete.
2. Click the delete icon at the right of the row.

> 3. Confirm device removal. Optionally, select whether to delete child devices of the device or to delete the asociated device owner (selecting both options is not supported).

Reviewer: ```suggestion
3. Confirm the device removal. Optionally, select whether to delete child devices of the device or to delete the associated device owner. Note that it is not possible to select both options.
```

Author: applied

> 3. Confirm device removal. Optionally, select whether to delete child devices of the device or to delete the asociated device owner (selecting both options is not supported).

Reviewer 2: ```suggestion
3. Confirm device removal. Optionally, select whether to delete child devices of the device or to delete the associated device owner. Selecting both options is not supported.
```

Author: changed to @MWindel's suggestion
